webapp/lib/controller.dart

```diff
@@ -84,53 +115,34 @@ class SuggestedRepliesCategoryData extends Data {
   }
 }
 
+// ====
 
-
-List<String> configurationSuggestedReplyLanguages;
 SuggestedRepliesManager suggestedRepliesManager = new SuggestedRepliesManager();
 String selectedSuggestedRepliesCategory;
-List<String> editedSuggestedReplies = [];
+Set<String> editedSuggestedReplyIds = {};
+
+TagManager tagManager = new TagManager();
+Set<String> editedTagIds = {};
 
 model.User signedInUser;
-final String selectedPackage = 'Change communications';
 
 void init() async {
   setupRoutes();
   view.init();
   await platform.init();
 }
 
-void initUI() {
-  router.routeTo(window.location.hash);
-
-  // Listener inits
-
-  platform.listenForSuggestedReplies(
-  (added, modified, removed) {
-    suggestedRepliesManager.addSuggestedReplies(added);
-    suggestedRepliesManager.updateSuggestedReplies(modified);
-    suggestedRepliesManager.removeSuggestedReplies(removed);
-
-    // Replace list of categories in the UI selector
-    (view.contentView.renderedView as view.PackageConfiguratorView).suggestedRepliesView.categories = suggestedRepliesManager.categories;
-    // If the categories have changed under us and the selected one no longer exists,
-    // default to the first category, whichever it is
-    if (!suggestedRepliesManager.categories.contains(selectedSuggestedRepliesCategory)) {
-      selectedSuggestedRepliesCategory = suggestedRepliesManager.categories.first;
-    }
-    // Select the selected category in the UI and add the suggested replies for it
-    (view.contentView.renderedView as view.PackageConfiguratorView).suggestedRepliesView.selectedCategory = selectedSuggestedRepliesCategory;
-    _populateReplyPanelView(suggestedRepliesManager.suggestedRepliesByCategory[selectedSuggestedRepliesCategory]);
-  });
-}
-
 void setupRoutes() {
   router = new Router()
     ..addAuthHandler(new Route('#/auth', loadAuthView))
-    ..addDefaultHandler(new Route('#/configuration', loadPackageConfigurationView))
+    ..addDefaultHandler(new Route('#/configuration', loadConfigurationSelectionView))
+    ..addHandler(new Route('#/configuration/tags', loadTagsConfigurationView))
+    ..addHandler(new Route('#/configuration/suggested-replies', loadSuggestedRepliesConfigurationView))
```

Comment:
Please see https://github.com/larksystems/Katikati-Core/issues/393

Reply:
thanks, I'll follow up in another PR with the rename